packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -356,6 +364,24 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
     return undefined;
   }
 
+  private _getFeedback(): JSX.Element | undefined {
+    if (this.props.feedback) {
+      return (
+        <div className="ComponentPage-feedbackSection">
+          <h2 className="ComponentPage-subHeading ComponentPage-variantsTitle" id="Feedback">
+            Feedback
+          </h2>
+          <Link href="https://github.com/OfficeDev/office-ui-fabric-react/issues/new/choose" target="_blank">
+            Submit feedback on Github
+          </Link>
+          {this.props.feedback}
```

Comment:
Agree with @cliffkoh - not sure what this is supposed to be. I think we should have just a boolean to start. I would have expected maybe a list of tags or a query for the github issues to show up here.